lxd/storage/storage.go

```diff
@@ -51,7 +51,8 @@ func InstanceImportingFilePath(instanceType instancetype.Type, poolName, project
 
 // GetStoragePoolMountPoint returns the mountpoint of the given pool.
 // {LXD_DIR}/storage-pools/<pool>
-// Deprecated, use GetPoolMountPath in storage/drivers package.
+//
+// Deprecated: Use GetPoolMountPath in storage/drivers package.
```

Comment:
Can we remove this function and update the callers to use GetPoolMountPath?

Reply:
There are no callers of `GetStoragePoolMountPoint()` but since it's exported we don't get notified about an unused function.

I'm not sure what to do in such cases. Is that OK to simply drop it since it's under `lxd/storage` which is not something other software should depend on?

Reply:
If its not used in microcluster or microcloud we can drop it as its not part of our public api